samples/java-protobuf-web-resources/src/main/proto/com/example/web_resources.proto

```diff
@@ -0,0 +1,36 @@
+// tag::file-service-api[]
+syntax = "proto3";
+package com.example;
+
+option java_outer_classname = "FileAction";
+
+import "google/api/annotations.proto";
+import "google/api/httpbody.proto";   // <1>
+import "kalix/annotations.proto";
+
+message File {
+  string file = 1;
+}
+
+message FileInDir {
+  string file = 1;
+  string directory = 2;
+}
+
+service FileService {
+  option (kalix.codegen) = {action: {}};
+  option (kalix.service).acl.allow = { principal: ALL };
+  rpc GetFile(File) returns (google.api.HttpBody) {   // <2>
+    option (google.api.http) = {   // <3>
+      get: "/{file}" // <4>
+    };
+  };
+
+  rpc GetFileInDir(FileInDir) returns (google.api.HttpBody) {
+    option (google.api.http) = {
+      get: "/{directory}/{file}"   // <5>
```

Comment:
This shows off that it can be done, but for actual use in a kalix service I expect you'd want to be able to map your components to JSON endpoints as well, so that the static page can interact with those.

For that I expect youd use some prefix `"/site/{directory}/{file}"` for the arbitrary resources, and rather than `"/{file}"` you'd want to explicitly map root, for a start page, so `"/"`.

Not sure if sample needs to reflect that or not though. Maybe a comment about that would be enough?

Reply:
Let me add a small json-request example to show off. I will also look at the changes you propose.

Reply:
I did that for `scala`. Let me know what you think and I will add the same for the `java` example.

Reply:
That looks good to me!

Reply:
👍 Let me align the java example.